### DISCUSSION TITLE
Repository not appearing as a github.io page

### DISCUSSION THREAD

**Discussion by TeddyBurnsides** [community/community#22989] [Other Feature Feedback, Questions, & Ideas]
<p>I currently have a user repository titled <strong><a href="http://theowiersema.github.io" rel="nofollow noopener">theowiersema.github.io</a></strong>.</p>
<p>From what I can tell from Github’s documentation, this should then viewable as a website at <a>theowiersema.github.io</a>. However, I keep getting 404 errors.</p>
<p>I have an index.html file in the main directory (it’s been there for a while now.)</p>
<p>This originally was viewable as a web site, but I made the mistake of changing the repository name which originally caused the web link to break, but changing it back to <a href="http://theowiersema.github.io" rel="nofollow noopener">theowiersema.github.io</a> (which it was originally) did not restore the web site functionality.</p>
<p>Any idea what I can do to resolve this?</p>

**Comment by TeddyBurnsides** [comment, marked as answer]
<p>Ignore this…i waited a bit longer and the pages appeared.</p>

**Comment by jamescaan20** [comment]
<p><a class="mention" href="/theowiersema">@theowiersema</a>  Thanks</p>
